### PR TITLE
fix: rework modular docs in PR #2139

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change?
https://github.com/eclipse/che-docs/pull/2139/

## What issues does this pull request fix or reference?
Modular docs.

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by max-cx** [comment]
@themr0c, :slightly_smiling_face:  could you please have a looks at these files if the modular docs make sense (first time I'm doing this, so not 100% sure). Thank you.

File: modules/administration-guide/pages/disabling-readiness-init-containers-for-olm-installer.adoc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[id="disabling-readiness-init-containers-for-olm-installer"]
```
Comment (themr0c) [inline modules/administration-guide/pages/disabling-readiness-init-containers-for-olm-installer.adoc]: Please align ID and file name to the title. Add an alias (redirection) if needed.
Title: contains attributes, such as `{prod-short}`
Id: only id friendly attributes, such as `{prod-id-short}`.
File name: no attributes, all lowercase with no spaces, example: `ref_a-che-example.adoc` (Notable exception: examples using the `{project-context}` attribute.

File: modules/administration-guide/partials/assembly_enabling-and-disabling-readiness-init-containers-for-olm-installer.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+:parent-context-of-enabling-and-disabling-readiness-init-containers-for-olm-installer: {context}
+
+[id="enabling-and-disabling-readiness-init-containers-for-olm-installer_{context}"]
+= Enabling and disabling the readiness init containers for the OLM installer
```
Comment (themr0c) [inline modules/administration-guide/partials/assembly_enabling-and-disabling-readiness-init-containers-for-olm-installer.adoc]: For consistency with the Installation Guide, please consider using "OperatorHub" rather than "OLM". See:

https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-on-openshift-4-using-operatorhub/

File: modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-olm-installer.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+.Procedure
+
+. Find the {prod-short} Operator subscription name:
++
+[source,shell,subs="+quotes,+attributes"]
+----
+$ {orch-cli} get subscriptions -n {prod-namespace}
+----
+
+. Find the CSV name in the {prod-short} Operator subscription:
```
Comment (themr0c) [inline modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-olm-installer.adoc]: Define CSV here rather than in line 28

File: modules/administration-guide/partials/assembly_readiness-init-containers.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
+:parent-context-of-readiness-init-containers: {context}
+
+[id="readiness-init-containers_{context}"]
+= Readiness init containers
+
+:context: readiness-init-containers
+
+{prod-short} Operator installs {prod-short} and starts its containers in the correct order. If a node with {prod-short} is restarted and all {prod-short} containers start simultaneously, some containers may fail because some other component they depend on is not ready. To avoid such failures, the readiness init containers queue the containers for {prod-short} components to start in the correct order.
```
Comment (themr0c) [inline modules/administration-guide/partials/assembly_readiness-init-containers.adoc:8]: If a node with {prod-short} is restarted => 
When one {orch-name} node that is running some {prod-short} workload becomes unavailable or restarts, {orch-name} redistributes the workload in random order.

File: antora.yml [hunk 5]
```diff
@@ -55,9 +55,9 @@ asciidoc:
     link-cli-github: https://github.com/che-incubator/chectl
     link-identity-provider-documentation-openshift-3: https://www.keycloak.org/docs/latest/server_admin/#openshift-3
     link-identity-provider-documentation-openshift-4: https://www.keycloak.org/docs/latest/server_admin/#openshift-4
-    link-identity-provider-github: "https://www.keycloak.org/docs/6.0//server_admin/#github"
-    link-identity-provider-oidc: "https://www.keycloak.org/docs/6.0/server_admin/#_oidc"
-    link-identity-provider-saml: "https://www.keycloak.org/docs/6.0/server_admin/#saml-v2-0-identity-providers"
+    link-identity-provider-github: "https://www.keycloak.org/docs/15.0//server_admin/#github"
```
Comment (mmorhun) [inline antora.yml:58]: How is this change related to the docs in this PR?

File: modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-olm-installer.adoc [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+$ {orch-cli} get subscription <subscription-name> -n {prod-namespace} -o yaml | grep installedCSV
+----
+
+. Edit the `ClusterServiceVersion` (CSV) YAML manifest:
++
+[source,shell,subs="+quotes,+attributes"]
+----
+$ {orch-cli} edit csv _<csv-name>_ -n {prod-namespace}
+----
+. Remove the following environment variable to (from) the Operator Deployment spec:
```
Comment (mmorhun) [inline modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-olm-installer.adoc]:
```suggestion
. Remove the following environment variable from the Operator Deployment spec:
```

File: antora.yml (hunk 5 above)
Comment (max-cx) [reply antora.yml:58]: @mmorhun It's just the outdated file that fails the links test. I can update this file as part of this PR so that we can have a built preview of the docs for this PR.

:) If you find a minute sometime this week to update your branch `che-20337` against upstream `master` at `/che-docs/`, that will also solve this problem.

Comment (mmorhun) [reply antora.yml:58]: I see, let's keep it then

File: modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-olm-installer.adoc (hunk 3 above)
Comment (max-cx) [reply modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-olm-installer.adoc]: @themr0c Thank you for catching this, Fabrice.
